CHANGELOG.md

```diff
@@ -43,6 +45,7 @@ Quality Control, and Behavioural Quality Control. (PR #6041)
 database (PR #5260)
 - New documentation for file permissions has been added to the README.md file. (PR #5323)
 - Dashboard study progression section performance improvement (PR #5887)
+documentation for file permissions has been added to the README.md file (PR #5323)
```

Comment:
I don't think this line is supposed to be here.. it's a duplicate of 2 lines above it (but the formating is also messed u..)

Reply:
good catch! I removed the line. Sorry about that...